scoap3/modules/search/static/templates/scoap3_search/facets.html

```diff
@@ -1,4 +1,4 @@
-<div ng-init="titles={'country': 'Country', 'journal': 'Journal'};limit=13">
+<div ng-init="titles={'country': 'Country / region / teritory', 'journal': 'Journal'};limit=13">
```

Comment:
Shouldn' it be bold ? Dunno for sure from issue

Reply:
Actually, no need to be bold, but we have to have them with title case "Country / Region / Territory"